servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/ContextPreservingBiConsumer.java

```diff
@@ -15,23 +15,17 @@
  */
 package io.servicetalk.concurrent.api;
 
-import io.servicetalk.context.api.ContextMap;
-
 import java.util.function.BiConsumer;
 
 import static java.util.Objects.requireNonNull;
 
 final class ContextPreservingBiConsumer<T, U> implements BiConsumer<T, U> {
-
-    // TODO: remove after 0.42.55
-    private final ContextMap saved;
```

Comment:
Bcz of this removal, consider waiting until we have one more release before merging this PR. Just to buy us a bit more time

Reply:
@idelpivnitskiy, did you mean 0.42.57 that was just released?